bloom-housing/bloom
# feat(backend): add order param to listings GET endpoint

# Pull Request Template

## Issue Overview

This PR addresses https://github.com/CityOfDetroit/bloom/issues/1091


## Description

This chagne adds `order` to listings query param and simplifies `getOrderByCondition`. This method no longer specifies multiple sort keys for no functional reason. 

## How Can This Be Tested/Reviewed?

Check if it's possible to sort listings in GET /listings API by orderBy param: name, status, unitsAvailable, application due date and open waitlist + order param "ASC" | "DESC"

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
